orchestrator/src/indexer/base.ts

```diff
@@ -44,7 +44,7 @@ export abstract class Indexer {
   }
 
   applyAuthorizationHeader(hostname: string): string | undefined {
-    if (ALLOWED_HOST.includes(hostname) && xTwitterInstance.isInitialized()) {
```

Comment:
Why remove this?
What if Twitter is not supported in the by this particular Oracle?

Reply:
i am using the bearer Token directly now. so no initialisation is needed